lms/templates/verify_student/incourse_reverify.underscore

```diff
@@ -0,0 +1,31 @@
+<div id="wrapper-facephoto" class="wrapper-view block-photo face-photo-step">
+  <div class="facephoto view">
+    <h3 class="title"><%- gettext( "Take Your Photo" ) %></h3>
+    <div class="instruction">
+      <p><%= gettext("Course key: ") + courseKey %></p>
+      <p><%= gettext("Checkpoint name: ") + checkpointName %></p>
+    </div>
+
+    <div class="wrapper-task">
+      <div id="webcam" class="task cam"></div>
+
+      <div class="wrapper-help">
+        <div class="help help-task photo-tips facetips">
+          <h4 class="title"><%- gettext( "Tips on taking a successful photo" ) %></h4>
+
+          <div class="copy">
+            <ul class="list-help">
+              <li class="help-item"><%- gettext( "Make sure your face is well-lit" ) %></li>
+              <li class="help-item"><%- gettext( "Be sure your entire face is inside the frame" ) %></li>
+              <li class="help-item"><%- gettext( "Can we match the photo you took with the one on your ID?" ) %></li>
+              <li class="help-item"><%- gettext( "Use the retake photo button if you are not pleased with your photo" ) %></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+
+<button id="submit"><%= gettext("Submit") %></button>
```

Comment:
The same id name should not appear twice on a page so can we make it more specific please?